src/op_mode/conntrack_sync.py

```diff
@@ -151,35 +143,53 @@ def xml_to_stdout(xml):
             call(f'sudo {conntrackd_bin} -C {conntrackd_config} -s exp')
             print()
 
-    elif args.show_status:
-        is_configured()
-        config = ConfigTreeQuery()
-        ct_sync_intf = config.list_nodes(['service', 'conntrack-sync', 'interface'])
-        ct_sync_intf = ', '.join(ct_sync_intf)
-        failover_state = "no transition yet!"
-        expect_sync_protocols = "disabled"
-
-        if config.exists(['service', 'conntrack-sync', 'failover-mechanism', 'vrrp']):
-            failover_mechanism = "vrrp"
-            vrrp_sync_grp = config.value(['service', 'conntrack-sync', 'failover-mechanism', 'vrrp', 'sync-group'])
-
-        if os.path.isfile(failover_state_file):
-            with open(failover_state_file, "r") as f:
-                failover_state = f.readline()
-
-        if config.exists(['service', 'conntrack-sync', 'expect-sync']):
-            expect_sync_protocols = config.values(['service', 'conntrack-sync', 'expect-sync'])
-            if 'all' in expect_sync_protocols:
-                expect_sync_protocols = ["ftp", "sip", "h323", "nfs", "sqlnet"]
+def show_status(raw: bool):
+    is_configured()
+    config = ConfigTreeQuery()
+    ct_sync_intf = config.list_nodes(['service', 'conntrack-sync', 'interface'])
+    ct_sync_intf = ', '.join(ct_sync_intf)
+    failover_state = "no transition yet!"
+    expect_sync_protocols = "disabled"
+
+    if config.exists(['service', 'conntrack-sync', 'failover-mechanism', 'vrrp']):
+        failover_mechanism = "vrrp"
+        vrrp_sync_grp = config.value(['service', 'conntrack-sync', 'failover-mechanism', 'vrrp', 'sync-group'])
+
+    if os.path.isfile(failover_state_file):
+        with open(failover_state_file, "r") as f:
+            failover_state = f.readline()
+
+    if config.exists(['service', 'conntrack-sync', 'expect-sync']):
+        expect_sync_protocols = config.values(['service', 'conntrack-sync', 'expect-sync'])
+        if 'all' in expect_sync_protocols:
+            expect_sync_protocols = ["ftp", "sip", "h323", "nfs", "sqlnet"]
             expect_sync_protocols = ', '.join(expect_sync_protocols)
```

Comment:
Should not this be one level to the left after changes?

Reply:
Again, not my code, but you are right. This conversion to string also should only be in the non-raw branch!